javascriptv3/example_code/libs/scenario/scenario.js

```diff
@@ -3,24 +3,38 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Prompter } from "./prompter.js";
-import { Logger } from "./logger.js";
-import { SlowLogger } from "./slow-logger.js";
+import { Prompter } from "../prompter.js";
+import { Logger } from "../logger.js";
+import { SlowLogger } from "../slow-logger.js";
 
-class Step {
+export class Step {
   /**
    * @param {string} name
    */
   constructor(name) {
     this.name = name;
   }
+
+  /**
+   * Alias for "name".
+   */
+  get key() {
```

Comment:
Why?

Reply:
The 'name' is used to index the context properties. I was originally going to pull the name from the step:
`Hi, ${c[getName.key]}! You are between the ages of...`
But, it's hard to read.
I'll get rid of it.